src/solver/JRLQPLeastSquareSolver.cpp

```diff
@@ -0,0 +1,219 @@
+/* Copyright 2017-2021 CNRS-AIST JRL and CNRS-UM LIRMM */
+
+#include <tvm/solver/JRLQPLeastSquareSolver.h>
+
+#include <tvm/scheme/internal/AssignmentTarget.h>
+
+#include <jrl-qp/utils/enumsIO.h>
+
+#include <iostream>
+
+namespace tvm
+{
+
+namespace solver
+{
+JRLQPLeastSquareSolver::JRLQPLeastSquareSolver(const JRLQPLSSolverOptions & options)
+: LeastSquareSolver(options.verbose().value()), status_(jrl::qp::TerminationStatus::UNKNOWN), autoMinNorm_(false),
+  big_number_(options.big_number().value()), damping_(options.damping().value())
+{
+  TVM_PROCESS_OPTION(maxIter, options_);
+  TVM_PROCESS_OPTION(warmStart, options_);
+  TVM_PROCESS_OPTION(logFlags, options_);
+  if(options.logStream())
+    options_.logStream(*options.logStream().value());
```

Comment:
Couldn't we simplify this by having a `jrl::qp::SolverOptions` field in `JRLQPLSSolverOptions`?

Reply:
And why separate `big_number` and `bigBnd` from `jrl::qp::SolverOptions` which are the same things afaik?